packages/react-pose-text/src/SplitText.tsx

```diff
@@ -37,27 +37,14 @@ const getPoseProps = (props: Props) => {
   return poseProps;
 };
 
-class SplitText extends PureComponent<Props, State> {
+class SplitText extends React.PureComponent<Props, State> {
   static getDerivedStateFromProps({ children }: Props, state: State) {
     return !state || children !== state.text ? parseText(children) : null;
   }
 
   props: Props;
-  Word: (props: PoseProps) => ReactElement<any>;
-  Char: (props: PoseProps) => ReactElement<any>;
-
-  constructor(props: Props) {
-    super(props);
-
-    this.state = {};
-
-    const { wordPoses, charPoses, children } = props;
-
-    parseText(children);
```

Comment:
seems like a leftover from the previous implementation, this function is pure and its result was unused here so it was safe to remove it (it was also safe to remove it as it's called inside `getDerivedStateFromProps` which gets called before EACH render, including the initial one, so there is no need to call it in the constructor)